UpdateProfile/__tests__/handler.test.ts

```diff
@@ -893,10 +895,56 @@ describe("UpdateProfileHandler", () => {
       expect.objectContaining({
         lastAppVersion: undefined
       })
-    )
+    );
     expect(result.kind).toBe("IResponseSuccessJson");
     if (result.kind === "IResponseSuccessJson") {
       expect(result.value.last_app_version).toBeUndefined();
     }
-  })
+  });
+
+  it.each`
+    givenProfile                   | model                                                 | is_reminder_enabled | expectedIsReminderEnabled
+    ${"without isReminderEnabled"} | ${aRetrievedProfile}                                  | ${undefined}        | ${false}
+    ${"without isReminderEnabled"} | ${aRetrievedProfile}                                  | ${false}            | ${false}
+    ${"without isReminderEnabled"} | ${aRetrievedProfile}                                  | ${true}             | ${true}
+    ${"with isReminderEnabled"}    | ${{ ...aRetrievedProfile, isReminderEnabled: false }} | ${undefined}        | ${false}
+    ${"with isReminderEnabled"}    | ${{ ...aRetrievedProfile, isReminderEnabled: false }} | ${false}            | ${false}
+    ${"with isReminderEnabled"}    | ${{ ...aRetrievedProfile, isReminderEnabled: false }} | ${true}             | ${true}
```

Comment:
Some scenario is missing:
```suggestion
    ${"with disabled isReminderEnabled"} | ${{ ...aRetrievedProfile, isReminderEnabled: false }} | ${undefined}        | ${false}
    ${"with disabled isReminderEnabled"} | ${{ ...aRetrievedProfile, isReminderEnabled: false }} | ${false}            | ${false}
    ${"with disabled isReminderEnabled"} | ${{ ...aRetrievedProfile, isReminderEnabled: false }} | ${true}             | ${true}
    ${"with enabled isReminderEnabled"}  | ${{ ...aRetrievedProfile, isReminderEnabled: true }}  | ${undefined}        | ${false}
    ${"with enabled isReminderEnabled"}  | ${{ ...aRetrievedProfile, isReminderEnabled: true }}  | ${false}            | ${false}
    ${"with enabled isReminderEnabled"}  | ${{ ...aRetrievedProfile, isReminderEnabled: true }}  | ${true}             | ${true}
```